consumer/Consumer.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <!-- <CodeAnalysisRuleSet>Consumer.ruleset</CodeAnalysisRuleSet> -->
```

Comment:
This was just for manual testing of overrides. I.e. if you remove the comment markup, so that the ruleset _is_ loaded, then you can see that you can override the settings that come with `Particular.CodeRules`, to make CA2000 an error.